### ISSUE TITLE
Maskable icon has white border around it

### ISSUE THREAD

**Issue by floogulinc** [GoogleChromeLabs/bubblewrap#253]
**Describe the bug**
The generated maskable icon has thin white border around it.

This seems to be the same bug from https://github.com/GoogleChromeLabs/bubblewrap/issues/240#issuecomment-660571328.

**To Reproduce**
I generated an APK using [PWABuilder](https://www.pwabuilder.com/) and provided both a regular icon and a maskable icon. This is the maskable icon provided:

<details>
<summary>maskable-512x512.png</summary>

![maskable icon](https://hydrus.app/assets/maskable-512x512.png)

</details>

**Expected behavior**
The maskable icon would display correctly cropped like when added to homescreen from Chrome.

**Screenshots**
Here are a couple of screenshots from the default launcher on my Pixel 3a comparing the app when added to homescreen from Chrome (left) and the generated APK (right):

Circle Icons | Rounded Rectangle Icons | Drop Icon
---|---|---
![image](https://user-images.githubusercontent.com/1300395/87929315-dbf75980-ca4b-11ea-8d44-3b48a07340ca.png) | ![image](https://user-images.githubusercontent.com/1300395/87929343-e6195800-ca4b-11ea-9da0-6e75fc71445c.png) | ![image](https://user-images.githubusercontent.com/1300395/87929359-eade0c00-ca4b-11ea-84b8-24357fa41218.png)

This behavior is also displayed in the app switcher but oddly enough the icon appears correctly in App Info but I think this is because it either crops it closer or uses the non-maskable icon.

**Smartphone (please complete the following information):**
 - Device: Pixel 3a
 - OS: Android 10
 - Browser: Chrome
 - Version 84

**Additional context**

The app url is: https://hydrus.app/
The web manifest is here: https://hydrus.app/manifest.webmanifest

**Comment by andreban** [comment]
It seems dropping the adaptive-icon padding by `0.25dp` (from `12.75dp` to `12.25dp`) would solve this. Something like this:

```xml
<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:aapt="http://schemas.android.com/aapt">
    <background>
            <layer-list>
                <item android:drawable="@android:color/white" />
                <item android:drawable="@mipmap/ic_maskable"
					android:top="12.25dp"
					android:right="12.25dp"
					android:left="12.25dp"
					android:bottom="12.25dp" />
            </layer-list>
    </background>
    <foreground android:drawable="@android:color/transparent" />
</adaptive-icon>

```

@NotWoods any views / ideas around this?

**Comment by floogulinc** [comment]
Looks like it could also help if an option was provided to specify the background color.

**Comment by JudahGabriel** [comment]
Just to +1 this bug, this appears to be the same issue I'm hitting.

<img src="https://user-images.githubusercontent.com/312936/87865123-1e267b00-c926-11ea-82d5-ae9f27168067.png" />

Here is my [manifest](http://messianicradio.com/manifest.json), [icon](https://messianicradio.com/images/chavah512x512.png), [maskable icon](https://messianicradio.com/images/chavah-maskable1000x1000.png), and [URL](https://messianicradio.com).

I'm seeing this bug on a Pixel 3a.

**Comment by NotWoods** [comment]
Decreasing the padding should be fine. Firefox uses the theme_color to set the background, that might be preferrable. Alternatively the edge pixels could be extended out.

**Comment by andreban** [comment]
This generates something very similar to the WebAPK output:
```xml
<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:aapt="http://schemas.android.com/aapt">
    <background>
        <layer-list>
            <item android:drawable="@android:color/white" />
            <item android:drawable="@mipmap/ic_maskable"
                android:top="8.5dp"
                android:right="8.5dp"
                android:left="8.5dp"
                android:bottom="8.5dp" />
        </layer-list>
    </background>
    <foreground android:drawable="@android:color/transparent" />
</adaptive-icon>
```

The screenshot bellow has a few apps with different icon configurations, with the icon generated by Bubblewrap the left and the WebAPK on the right:
![maskable-test](https://user-images.githubusercontent.com/1733592/88039020-f78f5c80-cb3e-11ea-8ce0-9ae3f8683d4f.png)

**Comment by floogulinc** [comment]
Thanks for the fix!

**Comment by andreban** [comment]
[`1.4.1`](https://github.com/GoogleChromeLabs/bubblewrap/releases/tag/v1.4.1) was released with the fix.

**Comment by JudahGabriel** [comment]
@andreban this should still be fixed even though you guys reverted support for SVG, yeah? 

I just got a report on PWABuilder about someone hitting this issue, and I'm wondering if the undoing the SVG support also undid this fix. One possibility is PWABuilder's Google Play platform was reverted to an ealier version of Bubblewrap - checking that on this side, but I first wanted to see if this bug was reintroduced by the SVG reversion.
